File: Problem Solving/Mini-Max Sum.cs
using System.CodeDom.Compiler;
using System.Collections.Generic;
using System.Collections;
using System.ComponentModel;
using System.Diagnostics.CodeAnalysis;
using System.Globalization;
using System.IO;
using System.Linq;
using System.Reflection;
using System.Runtime.Serialization;
using System.Text.RegularExpressions;
using System.Text;
using System;

class Solution {

    // Complete the miniMaxSum function below.
    static void miniMaxSum(int[] arr) {
        var ordered = arr.OrderBy(x => x);
        Console.WriteLine($"{ordered.Take(4).Sum(x => Convert.ToInt64(x))} {(long)ordered.Skip(1).Take(4).Sum(x => Convert.ToInt64(x))}");
    }

    static void Main(string[] args) {
        int[] arr = Array.ConvertAll(Console.ReadLine().Split(' '), arrTemp => Convert.ToInt32(arrTemp));
        miniMaxSum(arr);
    }
}

File: Practice/Problem Solving/Algorithms/Warmup/Mini-Max Sum.c
void miniMaxSum(int arr_count, int* arr)
{
    long sum = 0, max = 0, min = 1000000000;
    
    for (int i = 0; i < arr_count; i++)
    {
        sum += arr[i];
        
        if (arr[i] > max)
        {
            max = arr[i];
        }
        
        if (arr[i] < min)
        {
            min = arr[i];
        }
    }
    
    printf("%ld %ld", sum - max, sum - min);
}
